backend/src/app/rpc/mutations/teams.clj

```diff
@@ -359,12 +360,19 @@
               :role role))
       nil)))
 
+(def sql:upsert-team-invitation
+  "insert into team_invitation(team_id, email_to, role, valid_until)
+   values (?, ?, ?, ?)
+       on conflict(team_id, email_to) do
+          update set role = ?, valid_until = ?, updated_at = clock_timestamp();")
```

Comment:
in this case, is better to use `now()` I think.

The difference between `now()` and `clock_timestamp()` is that first is stable across transaction, and the second one always return the latest timestamp.

Reply:
fixed